### DISCUSSION TITLE
Request for change

### DISCUSSION THREAD

**Discussion by Dominoxan** [ITSpecialist111/ai_automation_suggester#25] [Ideas]
Hi!

Interesting integration, many thanks for that!

Not an issue but a .... suggestion:

Could you split notifications to separate items rather than 10 or more items combined? That would allow userd to dismiss some non interesting for them and leave those we would like to implement.

**Comment by ITSpecialist111** [comment]
Thank you! I've been exploring different ways to overcome the character limit in persistent notifications and attributes in Home Assistant. This may involve modifying the current custom integration or developing an addon (Docker container) to bypass these limitations.

Rest assured, this is definitely on the roadmap. My initial idea for wave 2 is to implement a swipe-to-acknowledge or dismiss feature, which would reduce the number of good/bad suggestions the integration provides. This will allow us to tailor the integration based on your individual preferences and dislikes.

**Comment by Senbei123** [comment]
Hey @ITSpecialist111, 
realy nice integration. Already got a few suggestion. There were not correct, but at least give me an idea, what I can do. 

I already got in contact a couple of times with the character limit in the persistent notifications. There I have the idea, to split the response from the AI model into different entities. E.g. get only the idea, and in another sensor the yaml-code!

Kind regards, Tristan
